package.json

```diff
@@ -22,10 +22,8 @@
   "devDependencies": {
     "@agoric/casting": "dev",
     "@agoric/ertp": "^0.14.2",
-    "@agoric/notifier": "^0.4.0",
     "@agoric/ui-components": "^0.2.33",
-    "@agoric/wallet": "^0.13.1",
-    "@agoric/web-components": "0.3.1-dev-8ac3c4c.0",
+    "@agoric/web-components": "^0.5.0",
```

Comment:
Will upgrade to 0.6.0 once https://github.com/Agoric/agoric-sdk/pull/6486 lands then the build should work

Reply:
A 0.6.0 wouldn't be published until a new release of the SDK is published.

I'd recommend you look into using prerelease versions that are automatically built and pushed to npm.